### PR TITLE
Updated version of SKY UX, Builder

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Waiting for Builder `rc.2` to be released, https://github.com/blackbaud/skyux-builder/pull/195

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,23 +29,26 @@
     "minify": "uglifyjs dist/bundles/stache.umd.js --screw-ie8 --compress --mangle --comments --output dist/bundles/stache.umd.min.js",
     "test": "skyux test"
   },
+  "peerDependencies": {
```
Comment (blackbaud-brandonhare) [inline package.json]: peer dependencies?

Comment (Blackbaud-SteveBrush) [inline package.json]: Yep. This module requires a certain module, but won't install it itself. This prevents version collisions when multiple modules need the same module.

Comment (Blackbaud-BrandonJones) [inline package.json]: 👍

**Comment by Blackbaud-BrandonJones** [comment]
I thought we also would need to update the Opaque token stuff as well as the renderer stuff (see TODOs and in affix-directive).   They are deprecated in angular 4?

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-BrandonJones Good call; I changed `OpaqueToken` to `InjectionToken`. I couldn't find the "todo" comment you were talking about, however.

**Comment by Blackbaud-SteveBrush** [comment]
https://github.com/blackbaud/stache2/issues/101

**Pull request merged**